src/core/lang.c

```diff
@@ -507,6 +507,8 @@ const uint8_t *lang_get_string(int group, int index)
                 return translation_for(TR_BUILDING_GOLD_MINE);
             case BUILDING_CITY_MINT:
                 return translation_for(TR_BUILDING_CITY_MINT);
+            case BUILDING_DEPOT:
+                return translation_for(TR_BUILDING_DEPOT);
```

Comment:
This is proof you won't need the above translation code.

Reply:
Done.